src/securityPolicy.c

```diff
@@ -239,7 +218,6 @@ security_policy_t policyForSignTxInit(
         uint8_t networkId,
         uint32_t protocolMagic,
         uint16_t numInputs MARK_UNUSED,
-        uint16_t numOutputs,
```

Comment:
I'd say we should either include all the numbers here (and mark them MARK_UNUSED if appropriate), or use just those that are relevant (I have a slight preference for this), i.e. we can remove numInputs from params.

Reply:
I absolutely on the side of removing unnecessary parameters.